js/apps/admin-ui/src/clients/authorization/policy/Regex.tsx

```diff
@@ -54,6 +55,34 @@ export const Regex = () => {
           {...register("pattern", { required: true })}
         />
       </FormGroup>
+      <FormGroup
+        label={t("targetContextAttributes")}
+        fieldId="pattern"
+        labelIcon={
+          <HelpItem
+            helpText={t("targetContextAttributesHelp")}
+            fieldLabelId="clients:targetContextAttributes"
+          />
+        }
+        isRequired
+        validated={errors.pattern ? "error" : "default"}
+        helperTextInvalid={t("required")}
+      >
+        <Controller
+          name="targetContextAttributes"
+          defaultValue={false}
+          control={control}
+          render={({ field }) => (
+            <Checkbox
+              id="required"
```

Comment:
This `id` should match the name of the field in the form.

```suggestion
              id="targetContextAttributes"
```

Reply:
checkbox's id does not matter but the controller name has to match with the fieldId of the formGroup. (tested locally)

Reply:
It does, closing this as resolved, but see my new review comment.